robosuite/environments/pick_place.py

```diff
@@ -399,31 +462,38 @@ def _load_model(self):
 
         lst = []
         for j in range(len(self.vis_inits)):
-            lst.append((str(self.vis_inits[j]), self.vis_inits[j]()))
-        self.visual_objects = lst
+            visual_ob = self.vis_inits[j](
+                name=("Visual" + self.item_names[j] + "0"),
+                joints=[], # no free joint for visual objects
+            )
+            lst.append((str(self.vis_inits[j]), visual_ob))
+        self.visual_objects = OrderedDict(lst)
 
         lst = []
         for i in range(len(self.ob_inits)):
-            ob = self.ob_inits[i]()
-            lst.append((str(self.item_names[i]) + "0", ob))
+            ob = self.ob_inits[i](
+                name=(self.item_names[i] + "0"),
+                joints=[dict(type="free", damping="0.0005")], # damp the free joint for each object
```

Comment:
is this for stability reasons?

Reply:
I mirrored this from the `PickPlaceTask` - I think we have always had that damping on the objects for this task